src/test/java/it/pagopa/pn/bff/pnclient/userattributes/PnUserAttributesClientImplTestIT.java

```diff
@@ -53,6 +53,74 @@ public void resetMockServerClient() {
         mockServerClient.reset();
     }
 
+    @Test
+    void getPgConsentByType() throws JsonProcessingException {
+        Consent consent = consentsMock.getPgTosConsentResponseMock();
+        String response = objectMapper.writeValueAsString(consent);
+        mockServerClient.when(request().withMethod("GET").withPath(path + "/TOS"))
+                .respond(response()
+                        .withStatusCode(200)
+                        .withContentType(MediaType.APPLICATION_JSON)
+                        .withBody(response)
+                );
+
+        StepVerifier.create(pnUserAttributesClient.getPgConsentByType(
+                UserMock.PN_UID,
+                CX_TYPE,
+                ConsentType.TOS_DEST_B2B
+        )).expectNext(consent).verifyComplete();
+    }
+
+    @Test
+    void getPgConsentByTypeError() {
+        mockServerClient.when(request().withMethod("GET").withPath(path + "/TOS"))
+                .respond(response().withStatusCode(404));
+
+        StepVerifier.create(pnUserAttributesClient.getPgConsentByType(
+                UserMock.PN_UID,
+                CX_TYPE,
+                ConsentType.TOS_DEST_B2B
+        )).expectError().verify();
+    }
+
+    @Test
+    void acceptPgTosConsent() throws JsonProcessingException {
+        String request = objectMapper.writeValueAsString(consentsMock.requestConsentActionMock());
+        mockServerClient.when(request().withMethod("PUT").withPath(path + "/TOS").withBody(request))
```

Comment:
Check the path if it is the same of the micro-service api

Reply:
done